.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,94 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Triage: Needed"]
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: happened
+    attributes:
+      label: What happened?
+      description: "Tell us what happened. Provide a detailed description of the issue with context."
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: What you expected?
+      description: "Tell us what you expected to happen."
+    validations:
+      required: true
+  - type: textarea
+    id: possible-fix
+    attributes:
+      label: Possible fix?
+      description: "Tell us about changes that resolve or influence the issue."
+    validations:
+      required: true
```

Comment:
this IMO should not be blocking requirement since not all people are devs or know what to do

Reply:
I've removed the validation.